src/service/resources-impl.js

```diff
@@ -262,6 +260,13 @@ export class ResourcesDef extends MutatorsAndOwnersDef {
    */
   removeForChildWindow(childWin) {}
 
+  /**
+   * @param {!Resource} resource
+   * @param {boolean} layout
+   * @param {number} parentPriority
```

Comment:
make this param optional.

also, try to refactor the param to `priorityOverride`, which is a more generic and meaningful param for a public method.